frontend/packages/console-app/src/components/cloud-shell/CloudShellTerminal.tsx

```diff
@@ -46,9 +46,9 @@ const CloudShellTerminal: React.FC<CloudShellTerminalProps> = ({ user, onCancel
   React.useEffect(() => {
     if (loaded && !loadError) {
       // workspace may be undefined which is ok
-      setCloudShellNamespace(workspaceNamespace);
+      setNamespace(workspaceNamespace);
```

Comment:
I would have to look into this further but looks like this is a change in behavior because `setCloudShellNamespace` only updated what was stored in localStorage. Now `setNamespace` also updates the state.

Unless absolutely sure, I suggest keeping the state namespace separate from the user settings.

Reply:
i tried verifying it and we had `setCloudShellNamespace` to set in localStorage and `getCloudShellNamespace` to get from LocalStorage so with `setNamespace` it'll update the ns in ConfigMap, don't we need that.

@sahil143 can you as well verify once?

Reply:
I tested it locally cloud shell seems to work as expected.

Reply:
Verified as well that setting this, while it does trigger an effect, will not cause any issues.